book/website/history.md

```diff
@@ -0,0 +1,43 @@
+# History of The Turing Way project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
+Dr. Whitaker and [close allies and contributors](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md#previous-project-members), who were already championing reproducible practices in data science and co-development the first version of *The Turing Way* book.
+The project team included researchers, librarians, data scientists and Research Engineers, from both The Alan Turing Turing and the institutes across the UK.
+The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working open.
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
```

Comment:
That is not the entire truth.
The goal of The Turing Way is to provide a 21st-century book -- not static, and something that allows interactivity and creativity. JupyterBook at the time was the only option that allowed combining executable notebooks with flat MarkDown files.
JupyterBook has since inspired Quarto's development that offers this feature.

I suppose the other reason that you are highlighting ties to our commitment to build on existing open source technology and ensure that we have the opportunity to contribute back to the Jupyter system that is a critical open source infrastructure.

Reply:
That's true. I think there might be an issue somewhere. @sgibson91 can probably share here as she was involved in the early days.

Reply:
Ok, so speaking with Kirstie:
- 21st-century element was priority as well as ensuring upstream, contributions
- Bookdown was not executable - and Jupyter was offering that solution
- Also, we had a close connection with the Jupyter Book team which could offer advice and support